root_github.tf

```diff
@@ -67,84 +67,6 @@ module "github_oidc_provider" {
   common_tags = local.common_tags
```

Comment:
The `locals` resource block on line 6 can be removed as its no longer used.

Reply:
Yeah, I have removed it and removed the `common_ssm_parameters` module as well, thanks